code/API_definitions/device-reachability-status-subscriptions.yaml

```diff
@@ -25,29 +24,45 @@ info:
     These endpoints allow to manage event subscription on reachability device status event.
     The CAMARA subscription model is detailed in the CAMARA API design guideline document and follows CloudEvents specification.
 
-    When subscribing, it is mandatory to provide the event `type` you are subscribing to, as multiple subscription-types are managed by this API.
+    When subscribing, it is mandatory to provide the event `types` you are subscribing to, as multiple subscription-types are managed by this API.
 
-    Following event ``type`` are managed for this API:
-      - ``org.camaraproject.reachability-status-subscriptions.v0.reachability-data``: Event triggered when the device is connected to the network for Data usage.
+    Following event ``types`` are managed for this API:
+      - ``org.camaraproject.device-reachability-status-subscriptions.v0.reachability-data``: Event triggered when the device is connected to the network for Data usage.
 
-      - ``org.camaraproject.reachability-status-subscriptions.v0.reachability-sms``: Event triggered when the device is connected to the network for SMS usage
+      - ``org.camaraproject.device-reachability-status-subscriptions.v0.reachability-sms``: Event triggered when the device is connected to the network for SMS usage
 
-      - ``org.camaraproject.reachability-status-subscriptions.v0.reachability-disconnected``: Event triggered when the device is not connected.
+      - ``org.camaraproject.device-reachability-status-subscriptions.v0.reachability-disconnected``: Event triggered when the device is not connected.
 
```

Comment:
It stated "reachability disconnected" but didn't specify whether it pertains to "SMS" or "data," leading us to assume it applies to both. So just confirming if there is no use-case to send the notification separately for each.

Reply:
As I understand it from the current specification, this `reachability-disconnected` - events occure when the device is not connected at all.
So you want to be informed when there is neither DATA nor SMS connection.

Please correct me if I am wrong @bigludo7

Reply:
I was reconfirming  if there is a separate event triggered when the device connects to the network for SMS and Data individually. Why do we only have one event triggered for disconnection?

Reply:
@maxl2287 Indeed as of now reachability-disconnected is when the mobile is not connected at all.

@sachinvodafone I got your point.
We have today 3 notification types: Data Connected, SMS connected and Disconnected. SMS connected mean that data is disconnected.
We have aligned this on the synchronous APIs response.

Alternatively we can define slightly distinct event as
- data-on 
- data-off
- sms-on
- sms-off

Max, Sachin wdyt?

Reply:
@bigludo7 can we either create a discussion on that topic or an issue?
Maybe a better place to discuss :)

Reply:
@maxl2287  Yes a discussion is a good idea. Do not block the PR for that. I copy/past las comment in a discussion.